velox/connectors/hive/tests/HiveDataSinkTest.cpp

```diff
@@ -664,6 +665,13 @@ TEST_F(HiveDataSinkTest, memoryReclaim) {
       {dwio::common::FileFormat::PARQUET, false, false, 1, false, false}
 #endif
   };
+  SCOPED_TESTVALUE_SET(
+      "facebook::velox::dwrf::Writer::MemoryReclaimer::reclaimableBytes",
```

Comment:
Why we need this? The threshold is set to 1GB in case of non-reclaimable?

Reply:
yeah need it to control releasable bytes